### DISCUSSION TITLE
v0.2.11

### DISCUSSION THREAD

**Discussion by HEIGE-PCloud** [HEIGE-PCloud/DoIt#63] [General]
### Enhancements
- 📜 replace smooth-scroll with native CSS b0bbe68 @HEIGE-PCloud (#7)
- 📖 improve readability for secondary text 0663843 @HEIGE-PCloud
- 📏 enhance the style of the horizontal rule 1eea238 @HEIGE-PCloud (#24)
- ✨ optimize for wide screen by removing the `max-width` limit 7d2a50b @HEIGE-PCloud (#27)
- 🔃 optimize lazyload for pictures b370f92 @HEIGE-PCloud (#43)

### New features
- 🖤 add black theme support fe1d58a @HEIGE-PCloud
- ✨ add [switch theme drop down menu](https://hugodoit.pages.dev/theme-documentation-basics/#site-configuration) e22750a @HEIGE-PCloud (#26)
- ✨ add support for [Waline comment system](https://waline.js.org) df8a9b6 @HEIGE-PCloud (#34)
- ✨ add [PWA support](https://hugodoit.pages.dev/pwa-support) 3a4f6be @HEIGE-PCloud (#56)
- ✨ add [new page style](https://hugodoit.pages.dev/theme-documentation-basics/#site-configuration) `wide` 3751b8b @HEIGE-PCloud (#35)
- 😎 add [friend shortcode](https://hugodoit.pages.dev/theme-documentation-extended-shortcodes/#12-friend) 997bb42 @HEIGE-PCloud (#31)

### Fixes
- 🐛 fix incorrect author name in RSS 79884a7 @zhengshihao.kirito
- 🐛 fix security issues with external links 8ff37f3 @HEIGE-PCloud (#19)
- 🐛 remove invalid Instagram shortcode b1948d8 @HEIGE-PCloud (#37)
- 🐛 check emptiness before inserting partials 527c0e3 @HEIGE-PCloud (#42)

### Chores
- ⬆️ update all third-party libraries (#59)

### Tests
- 🧪 add individual [test pages](https://hugodoit.pages.dev/categories/tests/) for all functions 27d16b6 @HEIGE-PCloud (#9)

[v0.2.10...v0.2.11](https://github.com/HEIGE-PCloud/DoIt/compare/v0.2.10...v0.2.11)

<hr /><em>This discussion was created from the release <a href='https://github.com/HEIGE-PCloud/DoIt/releases/tag/v0.2.11'>v0.2.11</a>.</em>

**Comment by DillonJoshua2004** [comment]
Oy send here, not in a link
________________________________
